frontend/sozisel-app/src/components/PresenterSession/Agenda/Agenda.tsx

```diff
@@ -0,0 +1,111 @@
+import "./Agenda.scss";
+
+import { Paper, Typography } from "@material-ui/core";
+import React, { ReactElement, useEffect, useState } from "react";
+
+import { AgendaPoint } from "../../../model/Agenda";
+import TurnedInNotIcon from "@material-ui/icons/TurnedInNot";
+
+interface AgendaProps {
+  agendaEntries: AgendaPoint[];
+  estimatedTime: number;
+}
+
+interface AgendaEntryProps {
+  startMinute: number;
+  endMinute: number;
+  name: string;
+  idx: number;
+}
+
+export default function Agenda({
+  agendaEntries,
+  estimatedTime,
+}: AgendaProps): ReactElement {
+  const [counter, setCounter] = useState<number>(0);
+  const agendaStatus = (startMinute: number, endMinute: number) => {
+    if (counter < startMinute) return "";
+    if (
+      (counter >= startMinute && counter < endMinute) ||
+      startMinute === endMinute
+    )
+      return "during";
```

Comment:
can you make an enum for that?

Reply:
Something like
```js
enum AgendaEntryStatus {
  NOT_STARTED,
  ONGOING,
  ENDED
}
```

Reply:
But these strings `during` etc... are used as a CSS class name which determined agenda header color, do we want to use CSS class name in capital letters in the scss file?

Reply:
At least make those variables a upper case constants at the top of the file.